1_html-css/cv.css

```diff
@@ -0,0 +1,258 @@
+

```

Comment:
I tried a lot, but I'm not sure if I did it true or not. because of no opportunity to viewing my weblog on my mobile phone.
so i hope what i did could work out.
please let me know my mistakes.